docs/writing-rules/experiments/join-mode/overview.md

```diff
@@ -196,7 +196,7 @@ Short for references, `refs` is a list of external rules that make up your code
 
```

Comment:
@minusworld on line 193 - <code>### `refs`</code> are these references now? I also don't see `refs:` in any example in this file.